### PR TITLE
Rust/may minihttp

### PR DESCRIPTION
**Pull request by enghitalo**

### CODE REVIEW

File: rust/may_minihttp/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[package]
+name = "server"
+version = "0.0.0"
+edition = "2021"
+
+[dependencies]
+may_minihttp = "0.1.11"
```
Comment (waghanza) [inline rust/may_minihttp/Cargo.toml:7]:
```suggestion
may_minihttp = "0.1"
```

File: rust/may_minihttp/config.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+framework:
+  github: Xudong-Huang/may_minihttp
+  version: 0.1.11
```
Comment (waghanza) [inline rust/may_minihttp/config.yaml:3]:
```suggestion
  version: 0.1
```

**Review by waghanza — COMMENTED** [review]
Thanks for your contribution @enghitalo 

We plan to include `may_minihttp` here. Are you OK with such a ting @Xudong-Huang ?

**Comment by Xudong-Huang** [comment]
> Thanks for your contribution @enghitalo
> 
> We plan to include `may_minihttp` here. Are you OK with such a ting @Xudong-Huang ?

Thanks! I'm total ok with that.

**Review by waghanza — APPROVED** [review]

**Comment by waghanza** [comment]
Results has been published @Xudong-Huang 

https://web-frameworks-benchmark.netlify.app/result

**Pull request merged**